wiki/Article_styling_criteria/Writing/de.md

```diff
@@ -351,16 +355,28 @@ Artikel sollten einfache deutsche Grammatik verwenden.
 
 ### Geviertstrich
 
-Geviertstriche (`—`) müssen mit einem Leerzeichen davor und dahinter verwendet werden. Diese Regel trifft nicht auf [Blockzitate](#blockzitate) zu.
+Geviertstriche (`—`) müssen mit einem Leerzeichen davor und dahinter verwendet werden. In [Blockzitaten](#blockzitate) haben Geviertstriche nur ein Leerzeichen auf ihrer rechten Seite. Bindestriche (`-`) dürfen nicht an Stelle von Geviertstrichen verwendet werden.
 
-### Blockzitate
+### Zitate
 
 Verweis auf [Wikipedias Artikel zum Format für Zitate](https://de.wikipedia.org/wiki/Wikipedia:Zitate#Beispiele).
 
 Füge abschließende Satzzeichen nur dann in die Anführungszeichen ein, wenn sie im Originaltext vorhanden waren und setze sie andernfalls nach dem schließenden Anführungszeichen. In den meisten Fällen bedeutet dies, dass Punkt und Komma genauso behandelt werden wie Fragezeichen: Sie stehen innerhalb der Anführungszeichen, wenn sie sich nur auf das zitierte Material beziehen, und außerhalb, wenn sie sich auf den gesamten Satz beziehen.
```

Comment:
This block is actually marked as a quote in the original (english) article, serving as the example that is referred to below. 
![image](https://user-images.githubusercontent.com/22643729/171900219-69114b22-891e-4da1-b6dc-59adda509cbd.png)
It's unfortunate that the german style guide linked here here doesn't feature an equivalent paragraph. Maybe we should add some other quote to serve as an example.

Reply:
you can mention in parentheses that the quoted material is translated from english, and also link to the english article

Reply:
```suggestion
Übersetzt aus dem englischen [Wikipedia Manual of Style](https://en.wikipedia.org
/wiki/Wikipedia:Manual_of_Style#Punctuation_inside_or_outside):

> Füge abschließende Satzzeichen nur dann in die Anführungszeichen ein, wenn sie im Originaltext vorhanden waren und setze sie andernfalls nach dem schließenden Anführungszeichen. In den meisten Fällen bedeutet dies, dass Punkt und Komma genauso behandelt werden wie Fragezeichen: Sie stehen innerhalb der Anführungszeichen, wenn sie sich nur auf das zitierte Material beziehen, und außerhalb, wenn sie sich auf den gesamten Satz beziehen.
```

If you want to follow the rule below about how to present sources it would have to be like this right?

Reply:
I think it's fine like this. Though, there is a line break in the link. Can you please remove it?

Another point is if we need to have links to both the German and English variant of the Wikipedia citation article 🤔

Reply:
I will apply the suggestion from @FamousPig as that is a bit shorter and additionally remove the reference to the German variant. Having both versions in the ASC may be confusing regarding what to use due to the differences between the two variants. If we ever need to specify more rules for citations (and thus change this section), we can do so later.